### PR TITLE
fix(datepicker): fixed DatePicker defaultPickerValue given number pan…

### LINKED ISSUE #735: DatePicker组件defaultPickerValue不生效
### Which Component 出现bug的组件
  - DatePicker

### semi-ui version
  - latest
    
### Expected result 期望的结果是什么
  - 设置defaultPickerValue后，组件渲染设置的时间

### Actual result 实际的结果是什么
  - 设置defaultPickerValue后，组件没有渲染设置的时间

### Steps to reproduce 复现步骤
  - 1 在DatePicker组件中加入属性：defaultPickerValue={[Date.now(), Date.now()]}

### Reproducible code 复现代码
```
class Demo extends React.component {

    render() {
        return (
            <DatePicker
               format="yyyy-MM-dd HH:mm"
               type="dateTimeRange"
               defaultPickerValue={[Date.now(),Date.now()]}
           />
        )
    }

}

```

### Additional information 补充说明
- 遇到这个bug的业务场景、上下文、或者你的需求场景

### PR DESCRIPTION
**Pull request by shijiatongxue**
…el render error #735

[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [x] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update
 - [ ] Refactor
 - [ ] Test Case
 - [ ] TypeScript definition update
 - [ ] Document improve
 - [ ] CI/CD improve
 - [ ] Branch sync
 - [ ] Other, please describe:

### PR description

Fixes #735 (bug from 0.x)

bug 见下图
image

### Changelog
🇨🇳 Chinese
- Fix: 修复 DatePicker defaultPickerValue 传数字时面板渲染错误问题 #735

---

🇺🇸 English
- Fix: Fixed the panel rendering error when DatePicker defaultPickerValue passes numbers #735

### Checklist
- [x] Test or no need
- [x] Document or no need
- [x] Changelog or no need

### Other
- [ ] Skip Changelog

### CODE REVIEW

**Comment by shijiatongxue** [comment]
defaultValue 和 defaultPickerValue 走的不是相同的处理函数

- defaultValue
image

- defaultPickerValue

image

File: cypress/integration/datePicker.spec.js [hunk 1]
```diff
@@ -266,4 +266,17 @@ describe('DatePicker', () => {
...
         }
     });
+
+    it('defaultPickerValue is number', () => {
+        cy.visit('http://localhost:6006/iframe.html?id=datepicker--fix-default-picker-value&viewMode=story');
+        cy.get('[data-cy=dateTime] .semi-input').first().click();
+        cy.get('[x-type=dateTime] .semi-datepicker-switch-text').first().contains('2021-03-15');
+        cy.get('[x-type=dateTime] .semi-datepicker-switch-text').last().contains('00:00:00');
+
+        cy.get('[data-cy=dateTimeRange] .semi-input').first().click();
```
Comment (DaiQiangReal) [inline cypress/integration/datePicker.spec.js:276]: 这里可以补充下 1970 年前的时间的测试用例

Comment (shijiatongxue) [reply cypress/integration/datePicker.spec.js:276]: 这些做的目的是？

Comment (shijiatongxue) [reply cypress/integration/datePicker.spec.js:276]: > 防止以后改代码在时间戳小于0的edge case 有bug

OK，我改下代码吧。既然要考虑小于 0 的话，就直接在不合法的时候用兜底的值。

Comment (shijiatongxue) [reply cypress/integration/datePicker.spec.js:276]: 已更新

**Review by DaiQiangReal — APPROVED** [review]

**Pull request merged**